returnn/datasets/meta.py

```diff
@@ -434,9 +437,8 @@ def _collect_single_seq(self, seq_idx):
     :rtype: DatasetSeq
     """
     seq_tag = self.seq_list_ordered[self.default_dataset_key][seq_idx]
-    features = self._get_data(seq_idx, "data")
-    targets = {target: self._get_data(seq_idx, target) for target in self.target_list}
-    return DatasetSeq(seq_idx=seq_idx, seq_tag=seq_tag, features=features, targets=targets)
+    features = {target: self._get_data(seq_idx, target) for target in self.target_list}
```

Comment:
This is very confusing. Why `self.target_list`? This is not supposed just the targets, but all features (I assume). So then use `self.data_keys`.

Reply:
Yes, that is true